companies/services/report_classifier.py

```
@@ -0,0 +1,112 @@
+                    company_type = "other"
+
+                logger.info(
+                    f"기업 유형 분류 완료: {company_name} → {company_type} ({reason})"
+                )
+                return company_type
+
+            logger.warning("JSON 파싱 실패, 기본값 'other' 사용")
+            return "other"
+
+        except json.JSONDecodeError as e:
+            logger.error(f"JSON 파싱 오류: {e}")
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)